libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/CriOAuthSessionService.java

```diff
@@ -17,25 +22,47 @@
 public class CriOAuthSessionService {
     private static final Logger LOGGER = LogManager.getLogger();
 
+    // Max sleeping time will be roughly WAIT_TIME * 2 ^ (MAX_ATTEMPTS - 1), plus execution time
+    // AWS say 'Consistency across all copies of data is usually reached within a second'
+    private static final int MAX_ATTEMPTS = 5;
+    private static final int WAIT_TIME_MILLIS = 50;
+
     private final DataStore<CriOAuthSessionItem> dataStore;
+    private final Sleeper sleeper;
```

Comment:
Curious why we use this sleeper class, when it's just wrapping Thread.sleep()?

Reply:
Just so we can stub it out in the tests I think